src/luv.c

```diff
@@ -508,6 +508,56 @@ static void luv_handle_init(lua_State* L) {
   lua_setfield(L, LUA_REGISTRYINDEX, "uv_stream");
 }
 
+// Call lua function, will pop nargs values from top of vm stack and push some
+// values according to nresults. When error occurs, it will print error message
+// to stderr, and memory allocation error will cause exit.
+LUALIB_API int luv_cfpcall(lua_State* L, int nargs, int nresult, int flags) {
+  int ret, top, errfunc;
+
+  // Get the traceback function in case of error
+  if ((flags & LUVF_CALLBACK_NOTRACEBACK) == 0)
+  {
+    lua_pushcfunction(L, luv_traceback);
+    errfunc = lua_gettop(L);
+    // And insert it before the function and args
+    lua_insert(L, -2 - nargs);
+    errfunc -= (nargs+1);
+  }else
+    errfunc = 0;
+  top  = lua_gettop(L);
+
+  ret = lua_pcall(L, nargs, nresult, errfunc);
+  switch (ret) {
+  case LUA_OK:
+    break;
+  case LUA_ERRMEM:
+    fprintf(stderr, "System Error: %s\n", lua_tostring(L, -1));
```

Comment:
these fprintf's on 534 and 544 should be wrapped as well.

Reply:
> should be wrapped
Sorry don't know your means, please give me an example.